java/README.md

```diff
@@ -1,188 +0,0 @@
-# docker-utils
```

Comment:
Why are we deleting documentation?

Reply:
The doc was mostly about the functionality that we deleted. Must be better to have it removed. Let me know if you want me to retain it.